pkgs/development/python-modules/thorlabspm100/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, unittestCheckHook
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "thorlabspm100";
+  format = "setuptools";
+
+  src = fetchPypi {
```

Comment:
I noticed during the build that the source fetched from pypi does not contain tests.
The check phase seems to work when fetched from github. Would you please change this?